### PR TITLE
v2

### PR DESCRIPTION
**Pull request by dhrubabasu**
- [x] Remove dependency on `DEPRECATED_GRAPH_URL`
	- [X] /pangolin/addresses
	- [x] /pangolin/transaction-average
	- [x] /pangolin/transaction-median (broken)

### CODE REVIEW

File: src/utils/constants.ts [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+export const TREASURY_VESTER_ADDRESS = '0x6747AC215dAFfeE03a42F49FebB6ab448E12acEe';
+export const COMMUNITY_TREASURY_ADDRESS = '0x650f5865541f6D68BdDFE977dB933C293EA72358';
+
+import {BigNumber} from '@ethersproject/bignumber';
+export const ONE_TOKEN = BigNumber.from('1000000000000000000');
+export const TOTAL_SUPPLY = ONE_TOKEN.mul(538000000);
+
+// https://github.com/pangolindex/interface/blob/master/src/state/stake/hooks.ts
+export const STAKING_ADDRESSES = [
+  '0x417c02150b9a31bcacb201d1d60967653384e1c6', // AVAX-ETH
```
Comment (dasconnor) [inline src/utils/constants.ts:19]: we have new addresses for the VSO token, you can get the address from my latest interface PR

File: tests/index.mjs [hunk 2]
```diff
@@ -0,0 +1,129 @@
+// eslint-disable-next-line node/file-extension-in-import
+import * as assert from 'uvu/assert';
+import {describe} from './setup/index.mjs';
+import {get} from 'httpie';
+
+const STAKING_ADDRESSES = [
+  '0x417c02150b9a31bcacb201d1d60967653384e1c6', // AVAX-ETH
```
Comment (dasconnor) [inline tests/index.mjs:7]: do you need to duplicate these constants? might need VSO here too

**Review by dasconnor — COMMENTED** [review]
Nice job, you definitely raised the quality of the codebase significantly

File: tests/index.mjs (hunk 2 above)
Comment (dhrubabasu) [reply tests/index.mjs]: Now importing from `src/utils/constants.ts` directly :)

**Pull request merged**